package.json

```diff
@@ -73,23 +73,25 @@
     "@babel/core": "7.2.2",
     "@babel/preset-env": "7.3.1",
     "@fortawesome/fontawesome-free": "5.7.1",
+    "@vue/server-test-utils": "1.0.0-beta.29",
+    "@vue/test-utils": "1.0.0-beta.29",
     "babel-loader": "8.0.5",
+    "babel-plugin-istanbul": "5.1.0",
     "chai": "4.2.0",
     "copy-webpack-plugin": "4.6.0",
     "css-loader": "0.28.11",
     "css.escape": "1.5.1",
     "emoji-regex": "7.0.3",
     "eslint": "5.13.0",
+    "eslint-plugin-vue": "5.1.0",
     "fuzzy": "0.1.3",
     "graphql-request": "1.8.2",
     "handlebars": "4.1.0",
     "handlebars-loader": "1.7.1",
     "html-minifier": "3.5.21",
     "html-minifier-loader": "1.4.1",
     "intersection-observer": "0.5.1",
-    "istanbul-instrumenter-loader": "3.0.1",
     "jquery": "3.3.1",
-    "jquery-ui": "1.12.1",
```

Comment:
🎉

Reply:
The new behavior is less good than the old one though, but I'm hoping we can fix that later.
Namely the drag also follows the x-axis, which is annoying, but not a big deal; also the content of the initial position gets duplicated. Not a big deal for channels but makes it ugly on whole networks. Not blocking merging this of course.